### PR TITLE
[Frontend] Integrate GET FAQ with Backend

### PR DESCRIPTION
**Pull request by amanbedi1**
## Issue that this pull request solves

 Closes: <a href="https://github.com/HITK-TECH-Community/Community-Website/issues/678">#678</a>

## Proposed changes
To fetch FAQs from API endpoint instead hardcoded

### Brief description of what is fixed or changed
Function added to fetch FAQs from API endpoint.

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

https://user-images.githubusercontent.com/82234871/147913566-56e34a88-2ff9-495f-9950-32d80631d450.mp4

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/pages/Faq/Faq.jsx [hunk 1]
```diff
@@ -37,172 +39,131 @@ export function Faq(props) {
...
-              </h3>
-            </AccordionSummary>
-            <AccordionDetails>
-              <Typography style={{ color: "white" }}>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-                Suspendisse malesuada lacus ex, sit amet blandit leo lobortis
-                eget.
+            {isFetching ? (
+              <Typography variant="h6" align="center" style={{color:"white"}}>
+                Please wait while we fetching FAQs
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Faq/Faq.jsx:120]: Please user loader instead of this, while fetching the data

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@amanbedi1 minor change required, rest looks good to me.

File: frontend/src/pages/Faq/Faq.jsx (hunk 1 above)
Comment (amanbedi1) [reply frontend/src/pages/Faq/Faq.jsx]: @Kajol-Kumari  made the required changes. Review it

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for making the requested changes @amanbedi1
LGTM 👍

**Pull request merged**